scripts/sgmcmc_subspace_lib.py

```diff
@@ -69,22 +71,31 @@ def subspace_optimizer(key, loglikelihood, logprior, params_init_tree, data, bat
 
 
 def subspace_sampler(key, loglikelihood, logprior, params_init_tree, build_sampler, data, batch_size,
-                    subspace_dim, nsteps_full, nsteps_sub, nsamples, use_cv=True,
-                    opt=optax.adam(learning_rate=0.1), pbar = True):
-    # Find good control variate / starting point in subspace
-    opt_key, sample_key = split(key)
-    params_tree, params_sub, log_post_trace, subspace_fns = subspace_optimizer(
-        opt_key, loglikelihood, logprior, params_init_tree, data, batch_size,
-        subspace_dim, nsteps_full, nsteps_sub, opt, pbar=pbar)
-    loglik_sub, logprior_sub, subspace_to_pytree_fn = subspace_fns
+                     subspace_dim, nsamples, opt=optax.adam(learning_rate=0.1),
+                     nsteps_full=0, nsteps_sub=0, projection_matrix=None, use_cv=True, pbar=True):
+    subspace_key, sample_key = split(key)
+
+    if nsteps_full > 0 and nsteps_sub > 0:
```

Comment:
I think this should be `if nsteps_full > 0 OR nsteps_sub > 0`

Reply:
Done